docs/examples/swift/iOS-CocoaPods/ThemisSwift/ThemisSwift.xcodeproj/project.pbxproj

```diff
@@ -428,7 +428,7 @@
 				SUPPORTS_MACCATALYST = NO;
 				SWIFT_OBJC_BRIDGING_HEADER = "$(SRCROOT)/$(PROJECT_NAME)/ThemisSwift-Bridging-Header.h";
 				SWIFT_SWIFT3_OBJC_INFERENCE = Default;
-				SWIFT_VERSION = 4.2;
+				SWIFT_VERSION = 5.0;
```

Comment:
I believe other example projects need to be updated to use Swift 5.0 as well.

Reply:
interestingly, Carthage example uses 5.0